1_prep.R

```diff
@@ -7,17 +7,18 @@ p1 <- list(
   # TODO - transfer to Denali using globus
   # list of lake-specific attributes for nml modification
   tar_target(p1_nml_list_rds, '1_prep/in/nml_list.rds', format = 'file'),
-  tar_target(p1_nml_lake_ids, names(readr::read_rds(p1_nml_list_rds))),
-  # lake - GCM cell crosswalk, filtered to only those lakes that are
-  # in nml list (able to be modeled by GLM)
-  tar_target(p1_lake_cell_xwalk_csv, '1_prep/in/lake_cell_xwalk.csv', format = 'file'),
-  tar_target(p1_lake_cell_xwalk_df, 
-             readr::read_csv(p1_lake_cell_xwalk_csv, col_types=cols()) %>%
-               filter(site_id %in% p1_nml_lake_ids)),
+  tar_target(p1_nml_site_ids, names(readr::read_rds(p1_nml_list_rds))),
+  # lake - GCM cell - GCM tile crosswalk, filtered to only those lakes
+  #  that are in nml list (able to be modeled by GLM)
+  tar_target(p1_lake_cell_tile_xwalk_csv, '1_prep/in/lake_cell_tile_xwalk.csv', format = 'file'),
```

Comment:
Might want to add a note here that this file is copied over from `lake-temperature-model-prep`